tslint.json

```diff
@@ -1,20 +0,0 @@
-{
```

Comment:
Why remove this file? Is this one not used in the project?

Reply:
replace tslint with eslint, becase tslint is deprecated